modules/Emails/EmailsController.php

```diff
@@ -722,7 +722,7 @@ protected function userIsAllowedToSendEmail($requestedUser, $requestedInboundEma
         }
 
         $isAllowedToUseOutboundEmail = false;
-        if ($outboundEmailAccount->type === 'system') {
+        if ($outboundEmailAccount->type === 'system' || $outboundEmailAccount->type === 'system-override') {
```

Comment:
@daniel-samson nice find, but it should be in the else if clause  (`user`), since when using system-override accounts `isAllowUserAccessToSystemDefaultOutbound()` will probaly return false. They're used to specify your own username and password, when the CRM does not allow users to use the system account after all.